SECURITY.md

```diff
@@ -1,5 +1,174 @@
-# Security Policy
+# Intro
 
-Sui is pre-release software under rapid development. It has not been audited and is not yet ready for production use.
+We appreciate your participation in keeping the Sui network secure. Please read all details on how to report an issue, what bug bounties are available, and who is eligible for bounties.
 
-After we have completed internal and external audits of Sui, we will establish a responsible disclosure policy and a bug bounty program covering both [protocol security](doc/src/learn/sui-security.md) and software security. In the meantime, please report security problems to security@mystenlabs.com.
+The program is run in partnership with [Immunefi](https://immunefi.com/). More information is available on [their site](https://immunefi.com/bounty/sui/).
+
+If you have any questions or concerns, please contact support@immunefi.com.
+
+# Reporting and response process
+
+All reports must be sent using Immunefi’s [secure dashboard](https://bugs.immunefi.com/). **Please DO NOT report a security issue using GitHub, email, or Discord.**
+
+You should receive an acknowledgement of your report within 48 hours for critical vulnerabilities and 96 hours for all other vulnerabilities.
+
+If you do not get a response, please contact support@immunefi.com. Please DO NOT include attachments or provide detail regarding the security issue.
+
+# Bug bounties
+
+The Sui Foundation offers bug bounties for security issues found at different levels of severity. If a vulnerability is found, please follow the process detailed above to report the issue. All bug reports must come with a testnet Proof of Concept with an end-effect impacting an asset-in-scope in order to be considered for a reward. You do not need to have a fix in order to submit a report or receive a bounty.
```

Comment:
We should provide a branch link to testnet-- @ebmifa is https://github.com/MystenLabs/sui/tree/testnet the right one?

Reply:
I updated this with the proper link--good callout Sam, and thanks for the confirmation, Eugene